src/react/components/pages/editorPage/editorPage.scss

```diff
@@ -122,7 +127,7 @@
             font-size: 80%;
 
             .asset-filename {
-                max-width: 175px;
+                max-width: 200px;
```

Comment:
Why this change?  Seems to cause some overflow in the left sidebar.
![image](https://user-images.githubusercontent.com/6540159/54220037-bb111280-44ad-11e9-80c8-da8ad9b4de3a.png)

Reply:
This was accidental, must have meant to edit another dimension. Good catch